pkg/discovery/discovery.go

```diff
@@ -21,13 +21,12 @@ import (
 )
 
 const (
-	// ExpectedPlayers is a maximum number of computation parties currently supported.
-	ExpectedPlayers = 2
 	mpcPodNameLabel = "mpc.podName"
 )
 
 var (
-	basePort        = int32(5000)
+	// BasePort is the base for the port number that is used by the proxy.
+	BasePort        = int32(5000)
```

Comment:
Is it required to change visibility of `BasePort`?

Reply:
This variable is now used in `spdz.go#getLocalPortForPlayer(...)`